### ISSUE TITLE
Regression test failing for Del Moral dataset on MySQL

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#269]
A regression test has just started failing on Travis for the Del Moral dataset on MySQL. E.g.;
https://travis-ci.org/weecology/retriever/builds/53258092

The tests are still passing locally for me and the PRs I merged today were all passing Travis when submitted, so I restarted some older builds that have been passing for extended periods of time and they are also now failing in the same way. I'm guessing that Travis has changed it's MySQL version, or something similar, resulting in a change in the hash.

**Comment by ethanwhite** [comment]
Interesting Travis is using the same SQL version (except for the Ubuntu part)

![screenshot from 2015-03-21 21 11 22](https://cloud.githubusercontent.com/assets/744427/6767858/155dff36-d00f-11e4-927a-d3ca7bc0d1e2.png)

as my local system

```
ethan@oryx:~$ mysql -u travis -Bse "SELECT @@version"
5.5.41-0ubuntu0.14.10.1
```

This test is passing fine on my local system.

**Comment by ethanwhite** [comment]
The test is failing because the bulk insert for the species table that is attempted by the MySQL engine is working on Travis, but not on my local machine. This means that the clean up steps in Engine.add_to_table get skipped for the species table because it is imported via mysql.insert_data_from_file. Since there are extra spaces in some of the family names, these don't get removed on Travis, but they do on my dev machine. This means that the dumps differ in the presence of the spaces and the test fails. Travis must have tweaked this configuration setting recently, which is what caused the test to start to fail.

**Comment by ethanwhite** [comment]
The fix for this should be to simply add

`*do_not_bulk_insert: True`

To the .script file under the `species` table.

This doesn't currently work. It appears that when the retriever is installed the `__init__` in `Script` is run twice (at least for DelMoral, probably for everything). The first time it is run it gets the passed in attributed in the `tables` dictionary correctly. The second time the attribute is missing and `tables` is overwritten without the necessary attribute.

@bendmorris - any thoughts about where this double calling of Script/BasicTextTemplates `__init__` is happening?

**Comment by ethanwhite** [comment]
Never mind. The script was getting installed twice because it was in both `~/.retriever/scripts/` and in the repository. Fix coming soon (hopefully).

**Comment by ethanwhite** [comment]
Fixed in a530e15c72e02db1b93210fe63fe0dc9b31d613f.
